src/PRo3D.Viewer/Properties/launchSettings.json

```diff
@@ -7,7 +7,7 @@
     },
     "PRo3D.Viewer.Provex": {
       "commandName": "Project",
-      "commandLineArgs": "--disableCors --port 4321 --remoteApi -loadRecent",
+      "commandLineArgs": "--disableCors --port 4321 --remoteApi --scene \"F:\\PRo3D\\AI-Mars-3D\\Co-Registration\\2025-01-07_CoRegDemo\\2025-01-07_Kodiak_CoReg_Demo_SRC_translated.pro3d",
```

Comment:
when using hardcoded paths, please use a specific launch setting

Reply:
it is not the best way either, but i started to do specific tests like this, leaving core startup launch settings clean: 
![image](https://github.com/user-attachments/assets/ebea2f20-01fc-4f60-bd66-3c9eb52d379c)